File: VB/AggregateFunctionCalcFieldExample/Form1.vb
Imports System.ComponentModel
Imports System.Drawing

Namespace AggregateFunctionCalcFieldExample

    Public Partial Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            reportDesigner1.OpenReport(New TestReport())
            reportDesigner1.ActiveDesignPanel.SelectedTabIndex = 1
        End Sub
    End Class
End Namespace

File: CS/AggregateFunctionCalcFieldExample/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace AggregateFunctionCalcFieldExample
{
    public partial class Form1 : DevExpress.XtraEditors.XtraForm
    {
        public Form1()
        {
            InitializeComponent();
            this.reportDesigner1.OpenReport(new TestReport());
            this.reportDesigner1.ActiveDesignPanel.SelectedTabIndex = 1;
        }
    }
}
